README.md

```diff
@@ -724,6 +758,31 @@ I18n.translations["pt-BR"] = {
 }
 ```
 
+## Using YAML file as config store
+Earlier versions of this gem was mainly using YAML config file for loading file.  
+And it was changed to have Ruby object as core configuration implementation.  
+For those using YAML config file and don't want to convert it into an initialzer,
+A loader class is created for loading such file.
+YAML config file will be supported for `3.x`, but might deprecated and removed in next major release.
+
+### Usage
+```ruby
+# Yes you still need some ruby code to load the YAML file,
+# but it should be easier than converting the whole YAML if it's large.
+
+require "i18n-js"
+::I18n::JS::Configuration::YamlFileLoader.new(
+  # Change this to be the path of your YAML config file
+  Rails.root.join("config", "i18n-js.yml"),
+).load
```

Comment:
I can add it afterward.
I want to avoid any automatic thing in this PR to so it safer for #262.
Reviewing more at once == bigger chance to miss something.
e.g. I forgot to append instead of overwriting `translation_segment_settings` when loading YAML file.